pygatt/backends/gatttool/gatttool.py

```diff
@@ -26,7 +26,7 @@
 if hasattr(bytes, 'fromhex'):
     # Python 3.
     def _hex_value_parser(x):
-        return bytearray.fromhex(x.decode('utf8'))
```

Comment:
@hbldh I had to change this to get the tests passing - `x` was not a string in the tests. This is probably just bad mock data in the test, since I think you verified this with actual hardware. I will make sure it works with hardware before committing this change.